spec/dummy/spec/integration/user_commands_spec.rb

```diff
@@ -18,4 +18,13 @@
       expect(result.error.messages[:name]).to include("can't be blank")
     end
   end
+
+  describe 'delete' do
+    it 'deletes record' do
+      users.create.call(name: 'Piotr')
+      result = users.try { delete(:by_name, 'Piotr') }
+
+      expect(result.error).to be(nil)
```

Comment:
ah, this should be like that:

``` ruby
rom.command(:users).try { delete(:by_name, 'Piotr') }
```

that should work...this spec was written before I introduced `try` interface so it grabs create command directly, which confused you. Sorry about that. I will update it now :) Thanks for reporting!